api/pkg/apis/v1alpha1/managers/activations/activations-manager.go

```diff
@@ -46,6 +50,11 @@ func (s *ActivationsManager) Init(context *contexts.VendorContext, config manage
 	} else {
 		return err
 	}
+	s.needValidate = managers.NeedObjectValidate(config, providers)
+	if s.needValidate {
+		s.Validator = validation.ActivationValidator{}
```

Comment:
This seems to be still redundant. Why we define NewActivationValidator(s.CampignLookup)?

Reply:
Fixed